runtimes/pythonrt/dockerclient.go

```diff
@@ -0,0 +1,298 @@
+package pythonrt
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/image"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/docker/docker/pkg/stdcopy"
+	"github.com/docker/go-connections/nat"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapio"
+)
+
+const (
+	runnersLabel       = "io.autokitteh.cloud.runner"
+	networkName        = "autokitteh_runners"
+	internalRunnerPort = "9293/tcp"
+)
+
+type dockerClient struct {
+	client          *client.Client
+	activeRunnerIDs map[string]struct{}
+	allRunnerIDs    map[string]struct{}
+	mu              *sync.Mutex
+	runnerLabels    map[string]string
+	logBuildProcess bool
+	logRunner       bool
+	logger          *zap.Logger
+}
+
+func NewDockerClient(logger *zap.Logger, logRunner, logBuildProcess bool) (*dockerClient, error) {
```

Comment:
When *don't* we want to log runner or build process?

Reply:
configurable for dev tests reduce logs